stories/Atom/Buttons-and-labels/Chips/chips.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { Chips } from './chips';
+import { Chipscross } from './chips_cross';
+
+export const getCaptionForLocale = (locale) => {
+  switch(locale) {
+    case 'english': const engText = {detail: 'Label'}; return engText;
+    case 'arabic': const arabicText = {detail: 'ملصق'}; return arabicText;
+    case 'burmese': const burmeseText = {detail: 'တံဆိပ်'}; return burmeseText;
+    case 'japanese': const japaneseText = {detail: 'ラベル'}; return japaneseText;
+    default:
+      return {detail: 'Label'};
+  }
+};
+
+<Meta title="Atoms/Buttons/Chips" component={Chips} />
+
+<style>{`
+    div#story--atoms-buttons-chips--chip {
+        margin-bottom: 30px;
+    }
+    .subheading {
+        font-weight: 700;
+        font-size: 18px;
+        color: #666;
+        letter-spacing: 1.5px;
+        line-height: 24px;
+        margin-bottom: 12px;
+        margin-top: 40px;
+    }
+`}</style>
+
+# Chips Component
+Chips can be used to represent small blocks of information.we can used either for contacts or for tags.
```

Comment:
@pallavikadam-srijan It does not give the proper info... @srijan-mehul-gada Please check